vars/generateAndExecutePipeline.groovy

```diff
@@ -119,13 +134,56 @@ def _getApplicationDefinitionFilePath() {
     return 'applicationDefinition.yaml'
 }
 
+def _validateReleaseBranch(applicationDefinition, scmMetadata) {
+
+    if (isReleaseBranch()==true) {
```

Comment:
Could we change this line by: if (isReleaseBranch()) { ....

Reply:
fixed